preview-env/comment/delete.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+## delete.sh
+##
+## Deletes the PRs summary comment
+## Background:
+## There's currently no well-maintained working github action out there, which we could use.
+## Thus we decided to reuse functions of `preview-env/clean/github.sh`
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+RELATIVE_SCRIPT_PATH=$(dirname "$0")
+COMMENT_BODY=${COMMENT_BODY?}
+PULL_REQUEST=${PULL_REQUEST?}
+
+# shellcheck source=/dev/null
+source "$RELATIVE_SCRIPT_PATH/../clean/github.sh"
```

Comment:
I was wondering if this would work 👍 
It's great, because we can export it to a more general folder next time.

Reply:
haha... my thought exactly. Since I wasn't sure if it works at all, i just "crapped" it here like it is now.
But I had the same idea of extracting this in a kind of "util" folder.